docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,58 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+Aliases allow you to define custom commands or change the default flag values for existing commands.
+
+New command `tsh l`:
+
+```yaml
+aliases:
+    "l": "tsh login --auth=okta --username=alice"
+```
+
+Make `tsh status` use JSON as a default format:
+
+```yaml
+aliases:
+    "status": "tsh status --format=json"
+```
+
+Define custom command using `bash`. The `$0` and `$1` variables will be substituted with command arguments.
+
+```yaml
+aliases:
+    "connect": "bash -c 'tsh login $0 && tsh ssh $1'"
+```
+
+The alias can use an arbitrary number of arguments. If a argument variable `$N` is referenced, `tsh` will check that at least `N+1` arguments were given to the alias invocation. All arguments that were given but not referenced in the alias definition will be appended at the end.
+
+Given configuration:
+
+```yaml
+aliases:
+    "example": "bash -c 'echo first=$0 $0-$1 $3'"
+```
+
+`tsh example 0 1 unused-2 3 unused-4` will expand to `bash -c 'echo first=0 0-1 3 unused-2 unused-4'`.
+
+Alias definition can also reference `$TSH` variable. It will be expanded to absolute path to current `tsh` executable. This can be useful if there are multiple `tsh` versions installed, or the currently used version is not in `PATH`.
```

Comment:
```suggestion
An alias definition can also reference the `$TSH` variable. `tsh` will expand this variable to the absolute path of the current `tsh` executable. This can be useful if there are multiple `tsh` versions installed, or the currently used version is not in `PATH`.
```